advanced_samples/CSV/dotnet/CSVtoOCS_pkce/azure-pipeline.yml

```diff
@@ -0,0 +1,140 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/advanced_samples/CSV/dotnet/CSVtoOCS_pkce'
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/advanced_samples/CSV/dotnet/CSVtoOCS_pkce'
+
+schedules:
+  - cron: '0 0 * * 0'
+    displayName: Weekly build
+    branches:
+      include:
+        - master
+    always: true
+
+variables:
+  - template: '/miscellaneous/build_templates/variables.yml'
+  - name: projPath
+    value: '/advanced_samples/CSV/dotnet/CSVtoOCS_pkce'
+  - name: testPath
+    value: '/advanced_samples/CSV/dotnet/CSVtoOCS_pkceTest'
+  - name: CredFile
+    value: appsettings.json
+  - name: CredFileLibrary
+    value: product-readiness.csvtoocs_dotnet.json
+  - name: covStream
+    value: CSVtoOCS_DotNet 
+  - name: bdProject
+    value: CSVtoOCS_DotNet
+
+jobs:
+  - job: Tests
+    strategy:
+      matrix:
+        # Cannot run on Linux due to missing Selenium Chrome Driver
+        # linux:
+        #   imageName: 'ubuntu-16.04'
+        windows:
+          imageName: 'vs2017-win2016'
+    pool:
+      vmImage: $(imageName)
+    steps:
+      - template: '/miscellaneous/build_templates/config.yml'
+        parameters:
+          fileOriginal: $(CredFile)
+          fileLibrary: $(CredFileLibrary)
+          filePath: $(projPath)
+
+      - script: |
+          dotnet restore
+          dotnet msbuild -p:HIGHENTROPYVA=true
+        workingDirectory: $(Build.SourcesDirectory)$(projPath)/
+        displayName: 'Build app'
+
+      - script: |
+          dotnet restore
+          dotnet msbuild -p:HIGHENTROPYVA=true
+          dotnet test
+        workingDirectory: $(Build.SourcesDirectory)$(testPath)/
+        displayName: 'Run tests'
+
+#manually added because there were issues with binskim
```

Comment:
What issues did you have with BinSkim? There is also already a parameter in the `analysis.yml` called `skipBinSkim` :)